firebase/firebase.py

```diff
@@ -243,8 +243,7 @@ def _build_endpoint_url(self, url, name=None):
             url = url + self.URL_SEPERATOR
         if name is None:
             name = ''
-        return '%s%s%s' % (urlparse.urljoin(self.dsn, url), name,
-                           self.NAME_EXTENSION)
+        return f'{urlparse.urljoin(self.dsn, url)}{name}{self.NAME_EXTENSION}'
```

Comment:
yes, more pythonic and the code returns the same result. We can prove with this small test:

```python3
url = "https://url.com/"
name = "john"
extension =".jpeg"
# Online Python - IDE, Editor, Compiler, Interpreter

print('%s%s%s' % (url, name,
extension) == (f'{url}{name}{extension}'))
```